docs/internal-documentation/maintenance/ReleaseProcess-deb-packages.md

```diff
@@ -4,28 +4,20 @@ Release Process - Create .deb Packages
 
 ## Overview
 
-The process to build the packages is split in two parts. First we update the
-"nightly packages", packages that are based on the develop branch of Zonemaster
-repositories. Then we promote those packages to "stable packages". The only
-difference between a nightly packages and its prometed version should be the
-upstream tarball. While the nightly packages use the develop branch as upstream,
-the stable ones use a specific tag.
+The purpose of this process is to create and publish .deb packages for
+Zonemaster components, currently only Zonemaster-CLI, Zonemaster-Engine and
+Zonemaster-LDNS are covered.
 
-Each package is split in two, one for the libraries and executable and an
-other one for the documentation. The documentation packages are marked as
-"Recommends" and will be installed by default alongside their corresponding
-package.
+The process to build the packages is split in two parts. First "nightly
+packages" are updated, those packages that are based on the develop branch
```

Comment:
too many words I think: `those packages that are based` -> `those packages are based`

Reply:
fixed